core/src/main/java/com/scalar/db/transaction/consensuscommit/MergedResult.java

```diff
@@ -0,0 +1,219 @@
+package com.scalar.db.transaction.consensuscommit;
```

Comment:
Is this expected to be here? I thought this file is added before.

Reply:
This class was originally an inter class in the `Snapshot` class. Because the change in this PR makes this class bigger, I separated it from `Snapshot`.